### PR TITLE
Feature adop 1973 dss part1

### PR DESCRIPTION
**Pull request by sairamkantheti1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java [hunk 1]
```diff
@@ -91,9 +91,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
...
     }
+
+    private void setDssMetaData(CaseData data) {
+
+        data.setDssQuestion1("Full Name");
+        data.setDssQuestion2("Date of Birth");
+        data.setDssQuestion3("Test");
+        data.setDssAnswer1("case_data.childrenFirstName, case_data.childrenLastName");
+        data.setDssAnswer2("case_data.childrenDateOfBirth");
+        data.setDssAnswer3("case_data.otherAdoptionAgencyAddress.Country");
```
Comment (yogendraU) [inline src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:108]: @sairamkantheti1 , is country an object or a string?

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java [hunk 2]
```diff
@@ -91,9 +91,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
...
             .data(data)
             .build();
     }
+
+    private void setDssMetaData(CaseData data) {
+
+        data.setDssQuestion1("Full Name");
+        data.setDssQuestion2("Date of Birth");
+        data.setDssQuestion3("Test");
+        data.setDssAnswer1("case_data.childrenFirstName, case_data.childrenLastName");
```
Comment (yogendraU) [inline src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:106]: If the backend has different fields for First name and last name and i believe, its been asked the same way while the case is created. Then DSS question should also ask for first name and last name separately. As if they are asked combined, we dont know if the data entered is Firstname first or last name first.

@YasharRahvar , correct me if i am wrong.

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 2 above)
Comment (YasharRahvar) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:106]: @yogendraU Standard way of entering full name is always first name followed by last name. If user enter last name first, then it is an incorrect answer. It's also an option for adoption team to make it clear to the user that they need to enter first name followed by last name in the question.
for example: What is applicant full name? (enter first name followed by last name)

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 1 above)
Comment (sairamkantheti1) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java]: @yogendraU 
Country is a string. Actually it is lower case country. I updated with same

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 2 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:106]: @sairamkantheti1 , could you please segregate first name and last name as two different questions

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 1 above)
Comment (sairamkantheti1) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java]: @yogendraU  Sure. I will make this change. Please update the same in comments of below user story including the exact question label
https://tools.hmcts.net/jira/browse/ADOP-1973

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 2 above)
Comment (sairamkantheti1) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:106]: @yogendraU  Sure. I will make this change. Please update the same in comments of below user story including the exact question label
https://tools.hmcts.net/jira/browse/ADOP-1973

Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java:106]: @sairamkantheti1 , i have updated the jira with comment

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java (hunk 2 above)
Comment (sairamkantheti1) [reply src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java]: @yogendraU made changes as per comments

**Review by yogendraU — DISMISSED** [review]

**Review by cshiwarale — APPROVED** [review]

**Review by yogendraU — APPROVED** [review]

**Pull request merged**